docs/mlir_interoperation.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>This feels very cluttered to me. If we want to show this, we should highlight the important parts (so <code>-t mlir</code>), otherwise this feels just too big IMO.  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Not sure, for what part this comment refers to?

Reply:
The xdsl-opt --help part. Not sure why the comment wasn't added there :thinking:

Reply:
I think the comment is at the right place when viewing it through the reviewnb website.